.github/workflows/ci.yml

```diff
@@ -91,6 +96,12 @@ jobs:
     name: CI for Prover Components
     uses: ./.github/workflows/ci-prover-reusable.yml
 
+  ci-for-zk-toolbox:
+    needs: changed_files
+    if: ${{ (needs.changed_files.outputs.zk_toolbox == 'true' || needs.changed_files.outputs.all == 'true') && !contains(github.ref_name, 'release-please--branches') }}
```

Comment:
We have to run it for core changes as well, otherwise there is chance, that they will broke init process